project/Settings.scala

```diff
@@ -8,7 +8,7 @@ import sbt.{Compile, Test, _}
 import sbtassembly.AssemblyPlugin.autoImport._
 
 object Settings {
-  lazy val artifactory = "https://artifactory.broadinstitute.org/artifactory/"
+  lazy val artifactory = "https://broadinstitute.jfrog.io/artifactory/"
```

Comment:
did you end up having to figure this out yourself? sorry we didnt communicate out that this was necessary. was such a pain for me to figure out haha.